cypress/e2e/cloud/annotations.test.ts

```diff
@@ -543,12 +543,18 @@ describe('The Annotations UI functionality', () => {
       cy.getByTestID('button')
         .click()
         .then(() => {
-          cy.getByTestID('selector-list schmucket').click()
-          cy.getByTestID(`selector-list m`).click()
+          // Look at this article for removing flakiness https://www.cypress.io/blog/2020/07/22/do-not-get-too-detached/
+          cy.getByTestID('selector-list schmucket')
+            .should('be.visible')
+            .click()
```

Comment:
I like the idea of adding `.should('be.visible')` but I think we should avoid chaining it. I'd recommend:
```
cy.getByTestID('selector-list schmucket').should('be.visible')
cy.getByTestID('selector-list schmucket').click()
```

Reply:
sure, im down with some non chaining, let me make the switch